inception-doc/src/main/resources/META-INF/asciidoc/user-guide/getting-started.adoc

```diff
@@ -1,86 +1,412 @@
+* For many questions, you find answers in the *main documentation*: <<{product-name} User Guide>>.
 
-Running {product-name} in the way you just did is a great way to get started and try out its
-capabilities, but it is not the best way of working with the application. If you like 
-{product-name},  please be sure to check out the <<admin-guide.adoc,Administrator Guide>> to 
-learn how to set up a production-ready instance.
+* You are welcome to join our *Google group* https://groups.google.com/forum/#!forum/inception-users[inception users].
 
-[NOTE]
-====
-By default, {product-name} creates and uses an embedded database and stores all its data in  folder called `.inception` (_dot inception_) within your home folder. While this allows you to get started very quickly in trying out the application, it is not a recommended configuration for serious use. For production use, please configure {product-name} to use a database server when using it in production. For more information, please refer to the <<admin-guide.adoc#sect_database,Administrator Guide>>.
-====
+* Or consider the {product-name} users *mailing list*: inception-users@googlegroups.com
+
+* You can also open an *issue on https://github.com/inception-project/inception/issues*[Github].
+
+
+
+=== Know Our Documentation for Further Reading
+Our main documentation consists of three parts: 
+
+* *User Guide*: If you only _use_ {product-name} and do not develop it, the xref:{product-name} User Guide[User Guide] beginning right after Getting Started is the one of your choice. If it does not answer your questions, don't hesitate to contact us (see <<Do You Have Questions or Feedback?>>). 
+
+NOTE: *User Guide-Shortcuts:* Whenever you find a *blue questionmark*-sign in {product-name}, you may click on it to be linked to the respective section of the User Guide.
+
+* *Admin Guide*: For more installation details, see the https://inception-project.github.io/documentation/latest/admin-guide[Admin Guide].
+
+* *Developer Guide*: {product-name} is open source. So if you would like to develop, the https://inception-project.github.io/documentation/latest/developer-guide[Developer Guide] might be interesting for you.
+
+
+== Installing and Starting INCEpTION
+
+=== Installing Java
+
+In order to run {product-name}, you need to have Java installed in version 8 or higher. If you do not have Java installed yet, please install the latest link:https://www.oracle.com/technetwork/java/javase/downloads/index.html[Oracle Java] or link:https://adoptopenjdk.net[OpenJDK]. Both work perfectly fine so you can choose either of them.
+
+
+=== Download and Start {product-name}
+
+In this section, we will download, open and log in to {product-name}:
+
+*Step 1 - Download:* Download the .jar-file from our
+https://inception-project.github.io/downloads/[website] by clicking on
+https://github.com/inception-project/inception/releases/download/inception-app-0.16.1/inception-app-standalone-0.16.1.jar[{product-name}
```

Comment:
I think, someone else requested to change it this this link but I'm not perfectly sure. So just for you three to know in case this point comes up again ;)